### PR TITLE
Syntax changes

### PR DESCRIPTION
**Pull request by nairihar**
I think passport.ts in config folder is not good idea, config folder for key value objects, passport init function like helper function, so i think need create helpers folder and put passport.ts in helpers folder, also in Helpers you can create function which will validate emails, and use in Auth.ts.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@ node_modules
 logs
 dist
 .env
-application.log
\ No newline at end of file
+application.log
+package-lock.json
```
Comment (davitovmasyan) [inline .gitignore]: @nairicode 

https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Comment by arturowin** [comment]
this.logger isn't available here. You can import winston and use winston.logger instead of this.logger .

File: src/middleware/guard.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const guard = (req: Request, res: Response, next: NextFunction) => {
         if (token) {
             jwt.verify(token, process.env.APPLICATION_SECRET, (err, user) => {
                 if (err) {
-                    console.log(err);
+                    this.logger.error(err.toString());
```
Comment (arturowin) [inline src/middleware/guard.ts]: this.logger isn't available here. You can import winston and use winston.logger instead of this.logger .

**Comment by arturowin** [comment]
@nairicode in general your improvements are good, well done.

File: src/middleware/guard.ts (hunk 2 above)
Comment (nairihar) [reply src/middleware/guard.ts]: oops had not seen this and thought logger is module from npm, just copy past

File: .gitignore (hunk 1 above)
Comment (nairihar) [reply .gitignore]: ok i will remove from .gitignore, just i have seen in other packages where package-lock.json in .gitignore
for example https://github.com/hapijs/hapi/blob/master/.gitignore#L17

**Pull request merged**